### PR TITLE
✨(app-desk) add modal add member

### PR DESCRIPTION
**Pull request by AntoLC**
## Purpose

Create button to add members, and integrate the modal with the routes.

## Proposal

- [x] feature members
- [x] routes `/teams/[teamId]/members/add`
- [x] integrate modal

### CODE REVIEW

File: src/frontend/apps/desk/src/pages/teams/[teamId]/members/add.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+const Page: NextPageWithLayout = () => {
+  return null;
+};
+
+Page.getLayout = function getLayout() {
+  return (
+    <TeamDetailLayout>
+      {(team) => <ModalAddMembers teamId={team.id} />}
+    </TeamDetailLayout>
```
Comment (lebaudantoine) [inline src/frontend/apps/desk/src/pages/teams/[teamId]/members/add.tsx:16]: I feel it's a bit overkill having a dedicated `url` to add members. If we look at other services (GitHub, Trello, drive, ...) The pop-in to add/invite users, is not linked to any `url`. What motivated creating a subpage?

File: src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+interface ModalAddMembersProps {
+  teamId: string;
+}
+
+export const ModalAddMembers = ({ teamId }: ModalAddMembersProps) => {
+  const { t } = useTranslation();
+  const router = useRouter();
+
+  return (
+    <Modal
```
Comment (lebaudantoine) [inline src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx:16]: We might have already discussed it, but I would recommend closing the modal on click outside. This behavior is common on most of the web service, if we take a look at Trello GitHub and Drive invitation, all modals close on click outside, and recover state when necessary on re-open. I think for the v1 we can close the modal on click outside, and ignore the recovering state part. What did motivate this choice? Johann's design doesn't have any closing icon, I guess, he wanted the modal to be closed when clicking outside.

**Review by lebaudantoine — COMMENTED** [review]
As there are cascade branches, let's discuss my comments and find the fastest way to ship code?

File: src/frontend/apps/desk/src/pages/teams/[teamId]/members/add.tsx (hunk 1 above)
Comment (AntoLC) [reply src/frontend/apps/desk/src/pages/teams/[teamId]/members/add.tsx]: I removed the commit and adapted the tests:
https://github.com/numerique-gouv/people/pull/125/commits/f4e8b49b97b287412c6fc3021e654e83c618283f#diff-2969e8751c72b7ce75b2eaac6c2a3899d4b34f439a128a13b48fb53737f38ad9R1

File: src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx (hunk 2 above)
Comment (AntoLC) [reply src/frontend/apps/desk/src/features/members/components/ModalAddMembers.tsx:12]: https://github.com/numerique-gouv/people/pull/125/commits/f4e8b49b97b287412c6fc3021e654e83c618283f#diff-1c926ca4662a2efb0ebca0a591e1243d1322baaab47ee82337e65b73a66974eeR20-R21

[(app-desk) close modal role on click outside](https://github.com/numerique-gouv/people/pull/125/commits/04f9843bf8f9274e19837c8e223de390c4dff62d)

**Review by lebaudantoine — APPROVED** [review]
LGTM

**Pull request merged**